.github/workflows/website-integrity.yml

```diff
@@ -0,0 +1,28 @@
+name: Website Integrity
+on:
+  pull_request:
+    paths:
+      - 'website/**'
+jobs:
+  website-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Use Node 16
+        uses: actions/setup-node@master
+        with:
+          node-version: 16
+          cache: 'yarn'
+      - name: Install Dependencies using Yarn
+        run: yarn --ignore-engines
+
+      - name: Build Packages
+        run: yarn build
+
+      - name: Typecheck
+        run: yarn workspace @graphql-yoga/website run build
+      - name: Diff
+        run: git diff --exit-code -- website/route-lockfile.txt
```

Comment:
#1058 ?

Reply:
This should actually always fail if there is a diff as you would want to generate this locally and commit it - or am I missing something

Reply:
hmm you're right, I wanted to do a git diff with exit 0/1 against the target branch

Reply:
so if somebody removed a redirect for example, we would compare route-lockfile.txt against master and see if something was modified/deleted (not added).

Reply:
pushed an update

Reply:
Would it be a lot of work to post the diff on the PR as a comment? 👀

Reply:
ok, done https://github.com/dotansimha/graphql-yoga/runs/5853908444?check_suite_focus=true